src/styles/partials/components/_breadcrumbs.scss

```diff
@@ -0,0 +1,73 @@
+.dg_breadcrumbs {
+  color: $black;
+  display: inline-block;
+  font-family: $default-heading-font;
+  font-size: $smallest-font-size;
+  font-weight: $font-bold;
+  list-style: none;
+  margin-bottom: 0px;
+  margin-top: $margin-small;
+  padding-bottom: $padding-smallest;
+  padding-left: 0px;
+  padding-top: 0px;
+  text-transform: uppercase;
+
+  i {
+    font-size: $h3-font-size;
+  }
+
+  ul {
+    margin-bottom: 0px;
+    padding-bottom: 0px;
+  }
+
+  li {
+    display: inline;
```

Comment:
can you change this to `inline-block` this will fix the height of the elements.

Reply:
Updated and committed.